sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -315,10 +315,10 @@ export class ServiceBusAdministrationClient extends ServiceClient {
     ruleExists(topicName: string, subscriptionName: string, ruleName: string, operationOptions?: OperationOptions): Promise<boolean>;
     subscriptionExists(topicName: string, subscriptionName: string, operationOptions?: OperationOptions): Promise<boolean>;
     topicExists(topicName: string, operationOptions?: OperationOptions): Promise<boolean>;
-    updateQueue(queue: QueueProperties, operationOptions?: OperationOptions): Promise<WithResponse<QueueProperties>>;
-    updateRule(topicName: string, subscriptionName: string, rule: RuleProperties, operationOptions?: OperationOptions): Promise<WithResponse<RuleProperties>>;
-    updateSubscription(subscription: SubscriptionProperties, operationOptions?: OperationOptions): Promise<WithResponse<SubscriptionProperties>>;
-    updateTopic(topic: TopicProperties, operationOptions?: OperationOptions): Promise<WithResponse<TopicProperties>>;
+    updateQueue(queuePropertiesWithResponse: WithResponse<QueueProperties>, operationOptions?: OperationOptions): Promise<WithResponse<QueueProperties>>;
```

Comment:
Do we have to rename the parameter as well?  If we definitely have to, then can we consider dropping "Properties"?

Reply:
Queue properties object is what they have.
`queueWithResponse` might add to the confusion?

I'm inclined to `queuePropertiesWithResponse` since it is probably the best translation of the type `WithResponse<QueueProperties>`.

Reply:
What about just `queue`?

Reply:
No strong feelings, can go with `queue`, the type and the JS docs would help the users anyway!

Reply:
Lets do that.

Reply:
Done!

Reply:
I dont see the API view updated though...

Reply:
Updated now :)